minesweeper.py

```diff
@@ -254,6 +270,7 @@ def flagging(event):
 button_image_pressed = PhotoImage(file=(image_dir / "pressed_button_small.png"))
 red_button_image_pressed = PhotoImage(file=(image_dir / "pressed_red_button_small.png"))
 flag_image = PhotoImage(file=(image_dir / "flag_small.png"))
+wrong_flag_image = PhotoImage(file=(image_dir / "wrong_flag_small.png"))
```

Comment:
wrong_flag_small is not the most beautiful image in the world. I made it for the logic's sake. We can decide together how we actually want it to look